### PR TITLE
Support gpu

### PR DESCRIPTION
**Pull request by KirillZubov**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
You need to take that gpu Project.toml as well.

**Comment by KirillZubov** [comment]
but `CUDA`  is already in Project.toml. Or what do you mean?

**Comment by ChrisRackauckas** [comment]
oh then you don't need the gpu environment.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 include: https://raw.githubusercontent.com/JuliaGPU/gitlab-ci/master/templates/v6.yml
 
+variables:
+  JULIA_CUDA_VERBOSE: 'true'
```
Comment (ChrisRackauckas) [inline .gitlab-ci.yml:4]: probably don't want that.

File: docs/src/solvers/pinn.md [hunk 2]
```diff
@@ -70,3 +70,7 @@ the subset of points from a full training set.
 Besides the high level of API: `discretize(pde_system, discretization)`, we can also use low-level API methods:  `build_loss_function`, `get_loss_function` ,`generate_training_sets`,`get_phi`, `get_derivative`.
 
 See how this can be used in docs examples or take a look at the tests.
+
+### GPUs
+
+Actually, it is just enough add `|>gpu` after `chain` and all will works.
```
Comment (ChrisRackauckas) [inline docs/src/solvers/pinn.md:76]: More formal. Describe it like, if the Chain is a GPU compatible chain then it's GPU accelerated. For Flux chains then this is done by doing ... For more information on using GPUs with Flux, consult ...

Comment (KirillZubov) [reply docs/src/solvers/pinn.md:76]: ok

**Comment by ChrisRackauckas** [comment]
looks like CI is failing?

**Comment by KirillZubov** [comment]
yeap, but isn't depend on GPU. It depends on sciml_train. We need add `progress = false` to `DiffEqFlux.sciml_train(...)` with ADAM opt.

**Comment by ChrisRackauckas** [comment]
how are there so many failures though? What happened?

**Comment by KirillZubov** [comment]
it need to add `progress = false` to  DiffEqFlux.sciml_train(loss, initθ, opt, progress = false; ) when we use `ADAM()`. It should solve the problem for `NNODE` and `NNRODE`.

**Pull request merged**